ultimatepython/classes/encapsulation.py

```diff
@@ -103,6 +104,14 @@ def main():
     account1.withdraw(50)
     assert (account1.get_balance() == 50)
 
+    # validating invalid amounts.
+    _ERROR_INPUTS = [-10, 0, 150]
+    for input in _ERROR_INPUTS:
+        try:
+            account1.withdraw(input)
+        except ValueError as e:
+            assert (str(e) in {_INSUFFICIENT_BALANCE_MESSAGE, _INVALID_AMOUNT_MESSAGE})
```

Comment:
Rename `_ERROR_INPUTS` to `error_inputs`. Also the assert does not need the outer-parenthesis wrapping.